docs/querying/querying.md

```diff
@@ -44,7 +49,7 @@ The Content-Type/Accept Headers can also take 'application/x-jackson-smile'.
 curl -X POST '<queryable_host>:<port>/druid/v2/?pretty' -H 'Content-Type:application/json' -H 'Accept:application/x-jackson-smile' -d @<query_json_file>
 ```
```

Comment:
Maybe add in a sentence here:
"If you are running quick start, you should replace <queryable_host>:<port> with localhost: 8082"

Sometimes I find it frustrating when reading tutorials but can't find the default parameter.

Reply:
Thank you for review!... About: 

>Can you double check the default ports used in the overall tutorial. Some places it uses 8888, but druid broker default seems to be 8082.

After trying them out with Druid in the quickstart config, both ports work, which makes sense, since it looks like the Broker is 8082 and the Router is at 8888. Properly though,  such requests should come through the router, no?